site/src/examples/carousel/index.module.css

```diff
@@ -1,8 +1,45 @@
-.carouselImagePlaceholder {
+.carousel {
+  --salt-text-h1-fontSize: 60px;
```

Comment:
Is this needed? We don't recommend our consumers doing this so it feels weird it being in an example